### ISSUE TITLE
[Servicing 3.1] [Accessibility] DefaultAction is empty for Combobox when "LegacyIAccessible" pattern is supported and Accessible Role is changed.

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#3713]
.NET Core Version: 3.1
Have you experienced this same bug with .NET Framework?: No

**Problem description:**
The DefaultAction is empty for Combobox when role is default and "LegacyIAccessible" pattern is supported.
The DefaultAction stays empty if change the Role property value and "LegacyIAccessible" pattern is supported.

**Expected behavior:**
The DefaultAction should be "Expand" for Combobox when role is default and "LegacyIAccessible" pattern is supported.
The DefaultAction property should be changed when Accessible Role is changed.

**Minimal repro:**
1. Add supporting of "LegacyIAccessible" pattern for Combobox
2. Create a .Net 5.0 app
3. Add a Combobox to the app form
4. Run the app and check "DefaultAction" property using Inspect

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang - could you please see if this bug repros on .NET 7.0. If it does not repro, I suggest we close this bug as we have no request for servicing. Thanks

**Comment by Olina-Zhang** [comment]
@Tanya-Solyanik  how to add supporting of "LegacyIAccessible" pattern for Combobox described in above step1? I verified it cannot repro in .Net core 3.1 and 7.0 if skip it.

**Comment by vladimir-krestov** [comment]
We can't set `LegacyIAccessible` outside. So, if it can't repro, let's close the issue.
